### DISCUSSION TITLE
Provide a long-lived download URL for all/new BombSquad releases

### DISCUSSION THREAD

**Discussion by Mrmaxmeier** [efroemling/ballistica#697] [General]
Hey,

efforts to package BombSquad for Linux distros ([AUR](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=bombsquad), [NixOS](https://github.com/NixOS/nixpkgs/pull/309886)) currently run into an issue with release downloads. The current version of the game is not available in the `/old/` directory, so they pull from `https://files.ballistica.net/bombsquad/builds/BombSquad_Linux_${_arch}_${_version}.tar.gz`. This URL goes bad after the next release, breaking package builds.

It'd be nice to have the current/new releases available both under `/builds/` and `/builds/old/` such that packaging scripts can use the long-lived `old/..` URL. (`archive/` might be a better name in this case :slightly_smiling_face:)
This might reduce pressure on the packagers to keep their versions up-to-date but is still a good change imo.

Thanks!

**Comment by SoK05** [comment]
Offtop, but good to see you're still here! You're a modding pioneer, man.

**Comment by ritiek** [comment]
Similar issue through [Lutris](https://lutris.net/games/install/36218/view).

**Comment by aryan02420** [comment]
similar issue with flatpak builds https://github.com/bombsquad02420/net.froemling.bombsquad/blob/main/net.froemling.bombsquad.yml#L27-L29

**Comment by Loup-Garou911XD** [comment]
> Hey,
> 
> efforts to package BombSquad for Linux distros ([AUR](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=bombsquad), [NixOS](https://github.com/NixOS/nixpkgs/pull/309886)) currently run into an issue with release downloads. The current version of the game is not available in the `/old/` directory, so they pull from `https://files.ballistica.net/bombsquad/builds/BombSquad_Linux_${_arch}_${_version}.tar.gz`. This URL goes bad after the next release, breaking package builds.
> 
> It'd be nice to have the current/new releases available both under `/builds/` and `/builds/old/` such that packaging scripts can use the long-lived `old/..` URL. (`archive/` might be a better name in this case 🙂) This might reduce pressure on the packagers to keep their versions up-to-date but is still a good change imo.
> 
> Thanks!

if a python script works for your case then here you go https://github.com/Loup-Garou911XD/bombsquad_server-termux/blob/main/get_latest_link.py

**Comment by MatiasMGz** [comment]
I hope efro fixes this. Yeah, i would like a more solid and long-live download URL for BombSquad versions!
